src/javascript/node/sessionless.d.ts

```diff
@@ -0,0 +1,16 @@
+
+export declare const AsyncFunction: (func: () => {}) => void;
+
+export declare const generateKeys: (getKey: () => {}, saveKey: () => {}) => Promise<{privateKey: string, publicKey: string}>;
```

Comment:
this should be `getKeys`, and `saveKeys`

Reply:
So here, and for the return of generateKeys above, you should define a Keys type that has privateKey: string, and publicKey: string